aries-site/src/pages/components/anchor.js

```diff
@@ -18,24 +24,154 @@ const Anchor = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+          designer="https://designer.grommet.io/anchor?id=HPE-design-system-hpedesignsystem-hpe-com"
+          docs="https://v2.grommet.io/anchor?theme=hpe#props"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/anchor/AnchorExample.js"
+          figma="https://www.figma.com/file/I7PsiUmvr7OEJ6311rBUfg/HPE-Anchor-Component?node-id=127%3A0"
+        >
+          <AnchorExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
         <SubsectionText>
-          Anchors are used with text based navigation, such as inline text,
-          header navigation, and footer navigation.
+          Anchors should be used when an element with a link is intended to
+          appear visually like a traditional browser link, text with a simple
+          underline.
         </SubsectionText>
-        <Example
-          details={[
-            `The default anchor color is brand, however, it is possible to use 
-            HPE Design System font colors for anchor in some use cases (i.e. 
-            header and footer). When using an inline text anchor, the color 
-            should remain default.`,
+      </Subsection>
+      <Subsection name="Usage of Anchor vs Button" level={3} gap="small">
+        <SubsectionText>
+          When deciding whether to use Anchor or Button, consider the visual
+          appearance of the element. If the element will look like a tradition
+          browser link, use Anchor. If there will be background or border
+          styling, use{' '}
+          <GrommetAnchor label="Button" href={nameToPath('Button')} />.
+        </SubsectionText>
+        <SubsectionText>
+          Semantically, if you apply an href to either an Anchor or a Button,
+          both will appear as an anchor tag in the DOM. This is why it is
+          important to consider the visual appearance of the element when
+          deciding whether to use Anchor or Button.
+        </SubsectionText>
+        <SubsectionText>
+          Here are some contexts that Anchor should be used:
+        </SubsectionText>
+        <BulletedList
+          items={[
+            'as inline text links',
+            'to add links to resources within contexts such as a form',
```

Comment:
Would it be helpful to display a variant showing anchor in a form context since it is specifically called out here?
Example: https://design-system.hpe.design/templates/forms#sign-up

Reply:
Yes, I think that could be a good addition.

Reply:
Done.